src/test/java/com/pngencoder/SubimageEncodingTest.java

```diff
@@ -100,7 +103,7 @@ public void testByteCustom() throws IOException {
         testImageEncoders(PngEncoderBufferedImageType.TYPE_CUSTOM, imgRGBA);
     }
 
-    private void validateImage(PngEncoderBufferedImageType type, BufferedImage image, PngEncoder encoder) throws IOException {
+    static void validateImage(PngEncoderBufferedImageType type, BufferedImage image, PngEncoder encoder) throws IOException {
```

Comment:
This looks pretty similar to https://github.com/pngencoder/pngencoder/blob/dc9967ecaf015812c665f0f4ae4c154b850847ce/src/test/java/com/pngencoder/PngEncoderTestUtil.java#L75
Maybe they can be consolidated?